src/components/dealInDetails/sliderAndTimerComponents/Store.tsx

```diff
@@ -187,7 +199,7 @@ const Store = () => {
         }}
       >
         <Typography sx={{ fontSize: theme.typography.subtitle1.xl }}>
-          {resultArray.length > 10 ? showAllText ? `${arr1} ${arr2}` : `${arr1}...` : testString}
```

Comment:
For the turncoat you can use css properties. Also MUI Typography  has option to auto truncate text. Maybe you have to adjust some css. or need to have max width.

Reply:
sir i will make that changes using css by tomorrow morning.